components/camera/replaypcd/replaypcd_test.go

```diff
@@ -167,30 +184,35 @@ func TestNextPointCloud(t *testing.T) {
 			endFileNum:   10,
 		},
 		{
-			description: "Calling NextPointCloud with bad source",
+			description: "Calling NextPointCloud batched no filter",
```

Comment:
Could we make a separate explicit test case for a batch that returns less than the requested batch size? (I know this one implicitly covers it since it uses batchSize2 and numPCDFiles is 15, but we could use batchSize3 here since the point of this one is to test no filter)